src/commands/entities/configure.ts

```diff
@@ -113,16 +113,16 @@ the corresponding property in the entity is not modified.
     } = options
 
     return {
-      anaphora: `ANAPHORA_${anaphora.toUpperCase().replace('-', '_')}`,
-      canonicalize: !noCanonicalize,
-      dataType: dataType.toUpperCase().replace('-', '_'),
+      ...(anaphora !== undefined && {anaphora: `ANAPHORA_${anaphora.toUpperCase().replace('-', '_')}`}),
+      ...(dataType !== undefined && {dataType: dataType.toUpperCase().replace('-', '_')}),
```

Comment:
These lines are getting long; consider refactoring these into variables. In `entities-helpers`, for example, you create a similar variable called `adjustedNewType`.

Reply:
Done.